### PR TITLE
Fix phone number serialization

### PR DESCRIPTION
**Pull request by maximrytych-ms**
Fixed the logic of PhoneNumberIdentifier to always maintain the original phone number string whether it included the leading + sign or not.

### CODE REVIEW

**Review by petrsvihlik — DISMISSED** [review]
please add changelog entries and version bumps for all touched packages except those that haven't been released since the introduction of the rawId (de)serialization.

File: sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py [hunk 1]
```diff
@@ -201,7 +198,7 @@ def identifier_from_raw_id(raw_id: str) -> CommunicationIdentifier:
     """
     if raw_id.startswith('4:'):
         return PhoneNumberIdentifier(
-            value='+{}'.format(raw_id[len('4:'):])
+            value='{}'.format(raw_id[len('4:'):])
```
Comment (AikoBB) [inline sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py:201]: do we really need formatting here? we can directly pass `raw_id[len('4:'):]` as a value

**Review by AikoBB — COMMENTED** [review]
why the jobrouter modality was excluded from the change?

File: sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py (hunk 1 above)
Comment (maximrytych-ms) [reply sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py]: Fixed.

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.3.1 (Unreleased)
+
+### Bug Fixes
+
+- Fixed the logic of `PhoneNumberIdentifier` to always maintain the original phone number string whether it included the leading + sign or not.
```
Comment (ostoliarova-msft) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:7]: As  `_shared` folder is copied across modalities, this change should be reflected in the changelog of other modalities (or at least those modalities that use `PhoneNumberIdentifier`)

**Review by AikoBB — APPROVED** [review]

File: sdk/communication/azure-communication-identity/CHANGELOG.md (hunk 2 above)
Comment (maximrytych-ms) [reply sdk/communication/azure-communication-identity/CHANGELOG.md:7]: We should only mention for modalities that expose this functionality to public. Job router does not expose this to public, so we don't have to mention it in their changelog. In other modalities we didn't change anything in the shared folder, so no need in mentioning it.

**Review by AikoBB — APPROVED** [review]

**Review by ostoliarova-msft — APPROVED** [review]

**Comment by JimSuplizio** [comment]
I'm going to override, the person who requested changes has been out and two others have approved it since then.

**Pull request merged**